traveler_app/src/main/res/layout/activity_profile.xml

```diff
@@ -2,104 +2,128 @@
-                    android:textSize="@dimen/text_view_small"
-                    android:textColor="@color/black"
-                    android:layout_width="match_parent"
-                    android:layout_height="match_parent"
-                    tools:text="gorlami_forever@guestlogix.com"/>
-        </LinearLayout>
-
+                android:clickable="false"
+                android:focusable="false"
+                android:gravity="center_vertical"
+                android:padding="@dimen/padding_small"
+                android:text="@string/saved_list"
+                android:textSize="@dimen/text_view_small"/>
     </LinearLayout>
 
-    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                  xmlns:tools="http://schemas.android.com/tools"
-                  android:orientation="horizontal"
-                  android:layout_width="match_parent"
-                  android:layout_height="wrap_content"
-                  android:id="@+id/layout_profileOrders"
-                  android:clickable="true"
-                  android:focusable="true">
+    <LinearLayout
+            android:id="@+id/layout_profileOrders"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="?android:attr/selectableItemBackground"
+            android:clickable="true"
+            android:focusable="true"
+            android:orientation="horizontal">
 
         <ImageView
                 android:id="@+id/imageView_profileOrders_icon"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center_vertical"
                 android:layout_margin="@dimen/padding_xlarge"
-                android:src="@drawable/ic_shopping_cart_c8_24dp"
                 android:clickable="false"
-                android:focusable="false"/>
+                android:contentDescription="@string/orders"
+                android:focusable="false"
+                android:src="@drawable/ic_shopping_cart_6a_24dp"/>
+
+        <TextView
+                android:id="@+id/textView_orders_title"
+                style="@style/TextButton.Profile"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:clickable="false"
+                android:focusable="false"
+                android:gravity="center_vertical"
+                android:text="@string/orders"
+                android:textSize="@dimen/text_view_small"/>
+    </LinearLayout>
+
+    <LinearLayout
+            android:id="@+id/linearlayout_email"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+        <ImageView
+                android:id="@+id/imageView_profileItem_icon"
```

Comment:
dont you have a convention on vies' names ? the best one is the same name used in java code and also starting with some kinda initial for the type of view. (e.g.: tvPassword, btnSend, imgProfile)

same naming in java and xml helps to have more readable code and migrating to kotlin would be easier (in kotlin views in xml and java code has the same name)